includes/views/upgrade-table.php

```diff
@@ -95,3 +96,48 @@
 		<td align="left" class="last">&nbsp;</td>
 	</tr>
 </table>
+
+<div class="upgrade-table-description">
+	<h1><?php esc_html_e( 'You might also like...', 'woocommerce-pdf-invoices-packing-slips' ); ?></h1>
+</div>
+<table id="upgrade-table">
+	<tr>
+		<td class="first feature-label">
+			<?php esc_html_e( 'WooCommerce Smart Reminder Emails', 'woocommerce-pdf-invoices-packing-slips' ); ?>
+			<?php echo isset( $all_plugins['wc-reminder-emails/wc-reminder-emails.php'] ) ? '<sup class="wcpdf_beta">' . __( 'Currently installed', 'woocommerce-pdf-invoices-packing-slips' ) . '</sup>' : ''; ?>
+			<br>
+			<span class="description">
+				<?php esc_html_e( 'Automatically schedule and send Reminder Emails for WooCommerce orders.', 'woocommerce-pdf-invoices-packing-slips' ); ?>
+				<a href="https://wpovernight.com/downloads/woocommerce-reminder-emails-payment-reminders/" target="_blank">
+					<?php esc_html_e( 'Visit plugin site', 'woocommerce-pdf-invoices-packing-slips' ); ?>
+				</a>
+			</span>
+		</td>
+	</tr>
+	<tr>
+		<td class="feature-label">
+			<?php esc_html_e( 'WooCommerce Print Address Labels', 'woocommerce-pdf-invoices-packing-slips' ); ?>
+			<?php echo isset( $all_plugins['woocommerce-address-labels/woocommerce-address-labels.php'] ) ? '<sup class="wcpdf_beta">' . __( 'Currently installed', 'woocommerce-pdf-invoices-packing-slips' ) . '</sup>' : ''; ?>
```

Comment:
Would it be better to change this text to "Activate it" if the plugin is installed but not activated?

Reply:
@MohamadNateqi I am not sure. Both ways make sense to me. Let's wait for @alexmigf and @Terminator-Barbapapa and their comments.

Reply:
I don't have a strong opinion, maybe @Terminator-Barbapapa has a preference.